resources/Microsoft.Windows.Setting.System/Microsoft.Windows.Setting.System.psd1

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+@{
+    RootModule           = 'Microsoft.Windows.Setting.System.psm1'
+    ModuleVersion        = '0.1.0'
+    GUID                 = 'cae11fd0-bf38-420f-9986-39d46990ae9b'
+    Author               = 'Microsoft Corporation'
+    CompanyName          = 'Microsoft Corporation'
+    Copyright            = '(c) Microsoft Corp. All rights reserved.'
+    Description          = 'DSC Module for Windows System Settings'
+    PowerShellVersion    = '7.2'
+    DscResourcesToExport = @(
+        'DeveloperMode'

```

Comment:
Should this be EmableDeveloperMode to match EnableRemoteDesktop and other recent settings?